zkg.meta

```diff
@@ -5,7 +5,7 @@ script_dir = build/scripts/Seiso/Kafka
 build_command = ./configure --with-librdkafka=%(LIBRDKAFKA_ROOT)s && make
 test_command = cd tests && btest -d
 plugin_dir = build
```

Comment:
so this will build tags?
We need to document the tagging then.

Reply:
It will make commits to the repo that update version references in zkg.meta, README, etc. and then tags that commit vX.Y.Z-rc1 and then when we push that the GitHub actions should automatically release.  Also facilitates rc2 or vX.Y.Z from there

Reply:
What you said there should be in the Maintainers.md

Reply:
See if you like what I wrote